### PR TITLE
Update input class for proper sizing

### PR DESCRIPTION
**Pull request by Dmac26**
﻿## Summary
Addresses Issue #1213      

This code update completes the 1213 card by updating all related .html, .ts, and .scss files to ensure that all numeric fields referenced in the 1213 card have their width reduced and validation updated to allow numeric ranges to be input with dashes included.

## This pull request is ready to merge when...
- [x] Feature branch starts with the issue number
- [x] Is connected to its original issue via zenhub 👇
- [x] All tests are passing and meet coverage, linting, and accessibility requirements. And no security vulnerabilities ⚫️(Circle)
- [ ] This code has been reviewed by someone other than the original author

### CODE REVIEW

File: frontend/src/app/application-forms/fields/activity-description.component.html [hunk 1]
```diff
@@ -42,7 +42,8 @@
     </ng-container>
 
     <input id="need-temporary-improvements" type="checkbox" formControlName="needTemporaryImprovements" />
-    <label id="need-temporary-improvements-label" class="usa-input" for="need-temporary-improvements">{{ pointOfView }} would like to use temporary directional signage.</label>
+    <label id="need-temporary-improvements-label" class="usa-input" for="need-temporary-improvements">{{ pointOfView }}
```
Comment (briandavidson) [inline frontend/src/app/application-forms/fields/activity-description.component.html:45]: This line break is unnecessary. Please revert it to maintain code style.

Comment (Dmac26) [reply frontend/src/app/application-forms/fields/activity-description.component.html]: @briandavidson I removed that line break yesterday

File: frontend/src/app/application-forms/validators/number-validation.ts [hunk 2]
```diff
@@ -1,14 +1,25 @@
...
-    const val = control.value;
-    if (val && val.length) {
+    if (allowDash) {
+      const val = control.value;
+      if (val && val.length) {
+        const numberRegex = /^[\d -]+$/;
+        const valid = numberRegex.test(val);
+        return valid ? null : { numberDashRequirement: true };
+      } else {
+      return null;
```
Comment (briandavidson) [inline frontend/src/app/application-forms/validators/number-validation.ts:12]: incorrect indentation here

File: frontend/src/app/application-forms/validators/number-validation.ts [hunk 3]
```diff
@@ -1,14 +1,25 @@
...
+    } else {
+      const val = control.value;
+      if (val && val.length) {
       const numberRegex = /^\d+$/;
       const valid = numberRegex.test(val);
       return valid ? null : { numberRequirement: true };
     } else {
       return null;
     }
+  }
```
Comment (briandavidson) [inline frontend/src/app/application-forms/validators/number-validation.ts:23]: The indentation around this `if` block needs to be adjusted.

Comment (Dmac26) [reply frontend/src/app/application-forms/validators/number-validation.ts]: @briandavidson - indentation adjusted and all excess .json files removed from the PR. Should be ready for a final review!

File: frontend/src/app/application-forms/validators/number-validation.ts (hunk 2 above)
Comment (mtlaney) [reply frontend/src/app/application-forms/validators/number-validation.ts:12]: Indentation should look something like this 

```
export function numberValidator(allowDash: boolean = false): ValidatorFn {
  return (control: AbstractControl): { [key: string]: any } => {
    if (allowDash) {
      const val = control.value;
      if (val && val.length) {
        const numberRegex = /^[\d -]+$/;
        const valid = numberRegex.test(val);
        return valid ? null : { numberDashRequirement: true };
      } else {
        return null;
      }
    } else {
      const val = control.value;
      if (val && val.length) {
        const numberRegex = /^\d+$/;
        const valid = numberRegex.test(val);
        return valid ? null : { numberRequirement: true };
      } else {
        return null;
      }
    }
  };
}
```

File: frontend/src/app/application-forms/fields/noncommercial-fields.component.html [hunk 4]
```diff
@@ -13,7 +13,7 @@
         <label id="participants-label" class="usa-input">Estimated number of participants <span class="required-fields-asterisk">*</span></label>
         <span id="participants-hint-text" class="help-text usa-form-hint">Please provide the number of event participants (Note: The use of a dash is permitted to supply a possible range).</span>
       </div>
-      <input class ="number-input-sizing" id="participants" type="tel" pattern="[0-9\-]+" maxlength="7" (keypress)="numberOnly($event)" formControlName="numberParticipants" aria-required="true" min="1"
+      <input class ="number-input-sizing" id="participants" type="tel" pattern="[0-9\-]+" maxlength="7" formControlName="numberParticipants" aria-required="true" min="1"
```
Comment (mtlaney) [inline frontend/src/app/application-forms/fields/noncommercial-fields.component.html:16]: input type should still be "text". we're not looking for a tel input. but the regex pattern is correct and will handle correct inputs. tested locally to work.

File: frontend/src/app/application-forms/fields/activity-description.component.html [hunk 5]
```diff
@@ -4,15 +4,15 @@
...
       [attr.aria-labelledby]="afs.labelledBy(parentForm.get('activityDescriptionFields.numberOfTrips'), 'number-of-trips-label', 'number-of-trips-error')"
       [attr.aria-invalid]="afs.hasError(parentForm.get('activityDescriptionFields.numberOfTrips'))"
      />
     <app-error-message fieldId="number-of-trips-error" name="Number of trips" [control]="parentForm.get('activityDescriptionFields.numberOfTrips')"></app-error-message>
 
     <label id="party-size-label" class="usa-input">The anticipated party size. <span class="required-fields-asterisk">*</span></label>
     <p id="party-size-hint-text" class="help-text usa-form-hint">If you are planning multiple trips or are unsure of the exact number of attendees, please provide a range that covers the smallest party size to largest party size. For example, for an operation that could have as little as 5 attendees but as many as 25 attendees, you could enter 5-25.</p>
-    <input class="usa-input" id="party-size"  type="text" formControlName="partySize" aria-required="true"
+    <input class="number-input-sizing" id="party-size"  type="tel" pattern="[0-9\-]+" maxlength="7" formControlName="partySize" aria-required="true"
```
Comment (mtlaney) [inline frontend/src/app/application-forms/fields/activity-description.component.html:15]: input type should still be "text". we're not looking for a tel input. but the regex pattern is correct and will handle correct inputs. tested locally to work.

File: frontend/src/app/application-forms/fields/activity-description.component.html [hunk 6]
```diff
@@ -4,15 +4,15 @@
 
     <label id="number-of-trips-label" class="usa-input">The anticipated number of trips. <span class="required-fields-asterisk">*</span></label>
     <span id="participants-hint-text" class="help-text usa-form-hint">Please provide the number of trips anticipated (Note: The use of a dash is permitted to supply a possible range).</span>
-    <input class ="number-input-sizing" id="number-of-trips"  type="tel" pattern="[0-9\-]+" maxlength="7"formControlName="numberOfTrips" aria-required="true"
+    <input class ="number-input-sizing" id="number-of-trips"  type="tel" pattern="[0-9\-]+" maxlength="7" formControlName="numberOfTrips" aria-required="true"
```
Comment (mtlaney) [inline frontend/src/app/application-forms/fields/activity-description.component.html:7]: input type should still be "text". we're not looking for a tel input. but the regex pattern is correct and will handle correct inputs. tested locally to work.

**Pull request merged**